source/main/resources/otc_fileformat/OTCFileformat.cpp

```diff
@@ -30,6 +30,11 @@
 #include <algorithm>
 #include <OgreException.h>
 
+RoR::OTCParser::OTCParser()
+{
+    m_def = std::make_shared<RoR::OTCFile>();
```

Comment:
We don't need this anymore, right?

Reply:
This is a mess, the object is created on 2 places: https://github.com/RigsOfRods/rigs-of-rods/pull/1849/files/4d600bc52ad894dd873cca59a64327260e39ba91#diff-fb9af0b3e9573c633daff2a2a4cba402R41